templates/definition/charger/nrgkick-bluetooth.yaml

```diff
@@ -1,3 +1,4 @@
+# note: this only works for older NRGkicks (before 2022/2023)
```

Comment:
rather put this into the description- the note is not visible in the docs

Reply:
Thanks for the hint! I have followed the structure based on the OCPP integration. Please let me know if it should be different.

Reply:
Ich dachte eher, das in den `Requirments` Text mit aufzunehmen. Für den Connect sollte sich m.e. auch nichts ändern, den gibts ja nicht sls neue Version, oder?

Reply:
Guter Punkt, ich ändere das noch.

Reply:
Ja, Konsistenz ist ein Hohes Gut…